docs/conceptual/snippets/fsarrays/snippet16.fs

```diff
@@ -1,3 +1,6 @@
 
-let multiplicationTable max = seq { for i in 1 .. max -> [| for j in 1 .. max -> (i, j, i*j) |] }
-printfn "%A" (Array.concat (multiplicationTable 3))
\ No newline at end of file
+Array.concat [ [|0..3|] ; [|4|] ]
+//output [|0; 1; 2; 3; 4|]
```

Comment:
Output should be placed in the doc instead of the code sample. I've included the links to the docs that reference this sample in the PR conversation thread.